### ISSUE TITLE
Webpack..?

### ISSUE THREAD

**Issue by viocassel** [uikit/uikit#4067]
### UIkit Version

*Check if the issue is reproducible with the latest stable version.*
3.2.5

### Actual Behavior

```
$ yarn compile
yarn run v1.21.1
$ yarn compile-less && yarn compile-js
$ yarn icons && node build/less
$ node build/icons
(node:5068) UnhandledPromiseRejectionWarning: Error: Could not resolve entry module (src\js\icons.js).
    at error (...\uikit\node_modules\rollup\dist\rollup.js:5351:30)
    at ...\uikit\node_modules\rollup\dist\rollup.js:12481:20
    at async Promise.all (index 0)
    at async Promise.all (index 0)
    at async Promise.all (index 0)
(node:5068) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async 
function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:5068) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not h
andled will terminate the Node.js process with a non-zero exit code.
```

Webpack was deleted in this release?
Unable to run test ...
And custom themes are not compiled.

```
$ yarn test
yarn run v1.21.1
$ webpack-dev-server --inline --hot --watchContentBase && open http://localhost:8080/tests/
"webpack-dev-server" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by janschoenherr** [comment]
Thanks, the icons task should be fixed with 3.2.6.

Webpack was removed, we no longer use it.

**Comment by viocassel** [comment]
Thanks @janschoenherr
